toolchain/check/eval.cpp

```diff
@@ -1333,6 +1333,10 @@ static auto TryEvalInstInContext(EvalContext& eval_context,
     case SemIR::CompleteTypeWitness::Kind:
       return RebuildIfFieldsAreConstant(
           eval_context, inst, &SemIR::CompleteTypeWitness::object_repr_id);
+    case SemIR::FacetValue::Kind:
+      return RebuildIfFieldsAreConstant(eval_context, inst,
+                                        &SemIR::FacetValue::type_inst_id,
```

Comment:
```suggestion
                                        &SemIR::FacetValue::type_id,
                                        &SemIR::FacetValue::type_inst_id,
```
The (facet) type of a facet value might involve symbolic constants, in which case we would need to update it too.